thrall/test/lib/elasticsearch/ElasticSearchTest.scala

```diff
@@ -29,9 +29,6 @@ class ElasticSearchTest extends ElasticSearchTestBase {
 
           val images: List[Image] = List(imageOne, imageTwo)
 
-          // in a clean index, we should have 0 documents
```

Comment:
If this causes the tests to pass in GHA and the test on line 41 still passes:

```scala
// after bulk inserting, we should have 2 documents
ES.client.execute(ElasticDsl.count(ES.initialImagesIndex)).await.result.count shouldBe images.length
```

It might suggest that force refreshing between tests might be a good thing to do?

Reply:
Yeah, that's a good idea.